src/main.rs

```diff
@@ -920,7 +925,6 @@ async fn start_ldk() {
 
 	// Stop the background processor.
 	bp_exit.send(()).unwrap();
```

Comment:
Will this work if the receiver has been dropped? Maybe condition this on `!bp_exit.is_closed()`.